chromadb/test/ef/test_spacy_ef.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from chromadb.utils.embedding_functions import SpacyEmbeddingFunction
+
+input_list = ["great work by the guy", "Super man is that guy"]
+model_name = "en_core_web_md"
+unknown_model = "unknown_model"
+
+
+def test_spacyembeddingfunction_isnotnone_wheninputisnotnone():
+    spacy_emb_fn = SpacyEmbeddingFunction(model_name)
+    assert spacy_emb_fn(input_list) is not None
+
+
+def test_spacyembddingfunction_throwserror_whenmodel_notfound():
+    with pytest.raises(ValueError,
+                       match="""spacy models are not downloaded yet, please download them using `spacy download model_name`, Please checkout
```

Comment:
We can add a portion of this text as `match` can do loose matching.

Reply:
Resolved